.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# What's new in this PR?
```

Comment:
I'd remove the `?` since it's not a question but a statement of what actually changed.

```suggestion
# What's new in this PR
```

Reply:
Done.